hentMiljøvariabler.sh

```diff
@@ -1,4 +1,4 @@
-kubectl config use-context dev-gcp
+kubectl config use-context nais-dev
```

Comment:
Jeg finner ikke nais-dev. 
![image](https://github.com/user-attachments/assets/a4f984e6-01a2-41d1-83a0-7b652b84203a)

Reply:
Dette med siste nais
➜  ~ brew upgrade nais
Warning: nais/tap/nais 1.20250130145505 already installed

Reply:
Og finner ingen info om at dev-gcp har byttet navn til nais-dev. SIkker på at du ikke bare skulle ha gjort dette:
https://nav-it.slack.com/archives/C01DE3M9YBV/p1734785029567539?thread_ts=1732701096.785659&cid=C01DE3M9YBV